examples/1App_SAS_SinglePin/README.md

```diff
@@ -0,0 +1,19 @@
+# 1App_SAS_SinglePin
+
+## Purpose
+
+This example provides a demonstration on how to use WATTS to model simple SAS model.
```

Comment:
The purpose here is exactly the same as the SodiumLoop one, so to me it wouldn't be clear what extra value the SodiumLoop one provides over this just based on the purpose alone

Reply:
Thanks for the comment @paulromano . I included the SodiumLoop example because it uses both the CHANNEL and PRIMAR4 modules from SAS while the SinglePin example only uses the CHANNEL module. Both modules produce separate output files that require different SAS utilities to be converted to CSV files. However, I do agree that there seems to be some repeated information between the two examples. I think we can either:
1.  Remove the SinglePin example
or 
2.  Change the purpose of the SinglePin example to "This example provides a demonstration on how to use WATTS to create a single-pin model with the CHANNEL module from SAS." and the purpose of the SodiumLoop example to "This example provides a demonstration on how to use WATTS to model a simple sodium loop with the CHANNEL and PRIMAR4 modules from SAS."
 
What do you think @nstauff ?

Reply:
@zhieejhia93 thanks for the comment. I think we can remove the SinglePin example. I will do so.